packages/x-tree-view-pro/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@mui/x-tree-view';
```

Comment:
Once we actually have a pro `SimpleTreeView` component, we won't be able to export everything from the root because of name collision (same as on the grid).
But for now this simplifies a lot the PR, and it will help us make sure we don't forget some export by looking at the export  JSON file.